Automattic-Tracks-iOS/Services/TracksService.m

```diff
@@ -42,6 +40,8 @@ @interface TracksService ()
 NSString *const DeviceInfoBrandKey = @"device_info_brand";
 NSString *const DeviceInfoManufacturerKey = @"device_info_manufacturer";
 NSString *const DeviceInfoModelKey = @"device_info_model";
+NSString *const DeviceInfoHeightPixelsKey = @"device_info_display_height";
+NSString *const DeviceInfoWidthPixelsKey = @"device_info_display_width";
```

Comment:
We could maybe remove "Pixels" from the consts since that had more to do with the (now obsolete) tracking pixel.

Reply:
Oh interesting 🤔 I thought of it more as identifying the units (i.e. pixels and not density-independent pixels) than related to the tracking pixel. Regardless makes sense to simplify, I'll update.

Reply:
(Done in 054595b.)